src/main/java/com/softserveinc/ita/pageobjects/DashboardPage.java

```diff
@@ -0,0 +1,4 @@
+package com.softserveinc.ita.pageobjects;
+
+public class DashboardPage extends MainMenu {
```

Comment:
as I understand, MainMenu as other pageobjects are admin pages, and probably need to create a separate admin package in pageobjects package